tests/testthat/test-mock-repos.R

```diff
@@ -64,7 +64,7 @@ test_that("repos, some basics", {
   expect_false(res$has_wiki)
 
   res <- gh(
-    TMPL("PATCH /repos/{owner}/{repo}"),
+    TMPL("patch /repos/{owner}/{repo}"),
```

Comment:
Not sure where to add a test of the lower-case support so I tweaked this one. :sweat_smile: